scripts/github-actions-build-docker-image.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -e # Turn on exit on error
+
+
+# Export environmental variables
+AWS_ACCOUNT_ID=$AWS_ACCOUNT_ID
+ECR_REPO_ENV=$ECR_REPO_ENV
+DEPLOYMENT_ENV=$DEPLOYMENT_ENV
+
+# Set default AWS region and tag
+
+export AWS_DEFAULT_REGION="us-east-1"
+export IMAGE_TAG="event-service"
+
+# assume github-runner-role
+OUT=$(aws sts assume-role --role-arn "arn:aws:iam::${ACCOUNT_NUMBER}:role/delegatedadmin/developer/github-actions-runner-role" --role-session-name github-actions);\
+export AWS_ACCESS_KEY_ID=$(echo $OUT | jq -r '.Credentials''.AccessKeyId');\
+export AWS_SECRET_ACCESS_KEY=$(echo $OUT | jq -r '.Credentials''.SecretAccessKey');\
+export AWS_SESSION_TOKEN=$(echo $OUT | jq -r '.Credentials''.SessionToken');
+aws sts get-caller-identity
+  
+            
+
+########### Logging into ECR ####################
+echo Logging in to Amazon ECR...
+aws ecr get-login-password --region $AWS_DEFAULT_REGION | docker login --username AWS --password-stdin $ACCOUNT_NUMBER.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com
+#export REPOSITORY_URI=$AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$ECR_REPO_ENV
+
+############# Building docker image ###############
+echo Build started on `date`
+echo Building the Docker image...          
+docker build --no-cache -t $ECR_REPO_ENV:$IMAGE_TAG .
+docker tag $ECR_REPO_ENV:$IMAGE_TAG $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$ECR_REPO_ENV:$IMAGE_TAG
```

Comment:
The way these images are tagged and handled, with all three services under different tags in the same "ab2d-prod-services" repo, is odd. We can leave it as is for now, but I've created [PLT-199](https://jira.cms.gov/browse/PLT-199) to address this issue.